secure-biometric-interface-integrator/lib/secureBiometricInterface.js

```diff
@@ -114,22 +121,44 @@ class SecureBiometricInterface {
   }
 
   /**
-   * Calling loader indicator
-   * @param {string} msg message to be shown in the loader
-   * @returns HTMLElement loader indicator
+   * Calling loading indicator
+   * @returns HTMLElement loading indicator with cancel button
    */
-  generateLoadingIndicator = (msg) => loadingIndicator(msg, this.isRtl);
+  generateLoadingIndicator = () => {
+    const elemArray = [
+      loadingIndicator(this.generateStatusMessage(), this.isRtl),
+    ];
+    if (this.status === states.LOADING) {
```

Comment:
could add a state for DISCOVERING

Reply:
fix in this commit [5ebb03c](https://github.com/mosip/esignet-plugins/pull/31/commits/5ebb03c7250691483cd36883d9cac477a1431126)